IdentityCore/src/network/request/MSIDAADBrowserPRTResponse.h

```diff
@@ -0,0 +1,22 @@
+//
+//  MSIDAADBrowserPRTResponse.h
+//  IdentityCore iOS
+//
+//  Created by Rohit Narula on 1/13/20.
+//  Copyright © 2020 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDAADBrowserPRTResponse : NSObject
```

Comment:
Does this need to live in common core or perhaps it belongs to broker?

Reply:
Addressed.